rebac-admin-backend/v1/swagger.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2024 Canonical Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	"net/http"
+
+	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+)
+
+// Returns the OpenAPI spec as a JSON file.
+// (GET /swagger.json)
+func (h handler) SwaggerJson(w http.ResponseWriter, req *http.Request) {
+	swagger, err := resources.GetSwagger()
+	if err != nil {
+		writeErrorResponse(w, NewUnknownError("cannot retrieve swagger instance"))
+		return
+	}
+
+	body, err := swagger.MarshalJSON()
+	if err != nil {
+		writeErrorResponse(w, NewUnknownError("cannot marshal spec as JSON"))
+		return
+	}
+
+	w.Write(body)
```

Comment:
would we want to log in case Write returns an error.. :shrug:

Reply:
Actually I had that in my mind, but we haven't yet selected a logging package/implementation for the library. I think for now, I can just use the standard logger and we can change it, if needed, in the future.